lib/data_migrate/config.rb

```diff
@@ -22,12 +22,20 @@ def initialize
       @data_template_path = DEFAULT_DATA_TEMPLATE_PATH
       @db_configuration = nil
       @spec_name = nil
+      @test_support_enabled = false
+      @test_framework = nil
```

Comment:
Nitpick:

It feels like the default shouldn't be nil, but we should attempt to infer rspec/minitest early. 

I'm not sure it makes as much sense from the "config lifecycle" perspective, though. If not, I wonder if we can make the limitation a bit more obvious?